scanners/whatweb/scanner/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2021 iteratec GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM debian:latest
+
+RUN apt-get update \
+    && apt-get install whatweb -y \
```

Comment:
I added the version tag and changed the Docker Container so that it now pulls from ruby and clones the git repository with the latest version tag. Unfortunately, there is no "latest" tag, so at the moment we have to manually update it..